source/testcommon/testcommon/__init__.py

```diff
@@ -0,0 +1,18 @@
+from testcommon.TestCases import TestCases
+from testcommon.read_csv import read_csv
+from testcommon.get_then_names import get_then_names
+
+# We are using the assertDataFrameEqual function from PySpark.
+# This function has A LOT of dependency issues. These have been resolved in
+# the setup.py file. So we can import the function here without any issues.
+from pyspark.testing.utils import assertDataFrameEqual as assert_dataframes
+from pyspark.testing.utils import assertSchemaEqual as assert_schemas
+
+
+__all__ = [
+    "TestCases",
```

Comment:
How about this concept, which ensures compile time checking?
```suggestion
    TestCases.__name__,
```

Reply:
There is nothing called compilation in Python. This is the standard way of doing it, and the way that spelling is ensures is by not having an unused import warning.

Reply:
I guess you can argue both ways 😄 
https://stackoverflow.com/questions/2998215/if-python-is-interpreted-what-are-pyc-files

In wonder why "my way" isn't best practice. It provides a better DX in PyCharm (which I use):

![image](https://github.com/user-attachments/assets/34b9263d-c14b-4baa-a978-051afc52d490)

Reply:
Not saying that it isn't best practice. Just saying that it's a little wierd and something that I've never seen before, neither in projects that I've worked in or any major open-source python project. That being said, I don't have strong opinions on this, so if you feel that it provides better DX, then let's go with your proposal.